src/components/MediaCard.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Loader from "react-loader-spinner";
+import Moment from "react-moment";
+
+function MediaCard(props) {
+  console.log(props);
+
+  function cardContent() {
+    if (!props.media) {
```

Comment:
I know this works, but I would actually separate the if conditionals here. 
I would keep the first one, and if it's the condition then return the loading screen. If it's not, then return the component's WHOLE return statement. 

Then inside that return statement, create a ternary operator, and check the media_type.

cleans the code up tremendously